### ISSUE TITLE
Multi-Cursor, Wrap lines, Selected Text and cursorLeft ==> cursors out of sync

### ISSUE THREAD

**Issue by rioj7** [microsoft/vscode#106398]
- VSCode Version: 1.49.0-insider  (2020-09-09T08:15  - e790b931)
- OS Version: Windows 8.1

Steps to Reproduce:

1. Create an html file with the following content
```
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8"/>
<title>Document</title>
</head>
<body>
<pre>python -m venv venv</pre>
<pre>.\venv\Scripts\Activate.ps1</pre>
<pre>Set-ExecutionPolicy Unrestricted</pre>
</body>
</html>
```
1. Enable Word Wrap
1. Size the window so we have a word wrap break in the first `</pre>` tag, splitting it in `</` and `pre>`, not in the other `</pre>` tags
1. Select the 3 `</pre>` tags
1. Move the 3 cursors to the beginning of the selections with `cursorLeft` <kbd>LeftArrow</kbd>
1. The cursors are out of sync. The selection that had a word wrap inside is moving 1 character before selection begin
1. Select all content inside `<pre>` tags with `Emmet: Balance (outward)`
1. Move the 3 cursors to the beginning of the selections with `cursorLeft` <kbd>LeftArrow</kbd>
1. The cursors are out of sync. The selection that had a word wrap inside is moving 1 character before selection begin

Does this issue occur when all extensions are disabled?: Yes

**Comment by rioj7** [comment]
* VSCode Version: 1.50.0-insider (2020-10-07T01:00 - 93c2f0f)
* OS Version: Windows 8.1

1. Create an HTML file with the following line:

```
<p>Lorem ipsum <span class="foobar">dolor sit amet consectetur</span> adipisicing elit. Iste, hic. Sequi eos facere <span class="foobar">voluptatibus dolore impedit</span>? Quae impedit a ut nobis est iure perspiciatis, fugit dignissimos quis doloribus voluptates odit? <span class="foobar">Quibusdam temporibus minus</span> odio et. Explicabo recusandae minus at, ullam id dolore maxime magni <span class="foobar">alias voluptas</span>, iure expedita animi veritatis.</p>
```

2. Enable Word Wrap

3. Size the window so we have a word wrap break in the second `<span class="foobar">` tag, splitting at `span ` and `class`, not in the other `<span class="foobar">` tags

4. Place the cursor at the end of the second `<span class="foobar">` tag before the word `voluptatibus`

5. Select `<span class="foobar">` with <kbd>Shift</kbd>+<kbd>LeftArrow</kbd>

6. Select the next 2 instances with <kbd>Ctrl</kbd>+<kbd>D</kbd>

7. Go to the end of the selections with <kbd>RightArrow</kbd>

8. Cursors are out of sync. The first selection is 1 character beyond the `>`

**Comment by hediet** [comment]
This issue seems to be fixed in VS Code 1.56:

![1d447fd8-350f-467b-9760-79e082a97060](https://user-images.githubusercontent.com/2931520/117308993-50d43c80-ae82-11eb-93f6-6d08b6262046.gif)

Please reopen if you can still reproduce this issue.
